cbioportal_etl/steps.py

```diff
@@ -34,6 +34,24 @@ def fetch_validator_scripts(tool_dir: str) -> None:
             raise
 
 
+def fetch_dgd_fusion() -> None:
+    """Download fusion-dgd.tsv.gz if it doesn't already exist in cwd"""
+    filename: str = "fusion-dgd.tsv.gz"
```

Comment:
We might want to make these params  that are pulled form config in case the file name and/or location changes